lib/plug/conn.ex

```diff
@@ -1504,7 +1632,7 @@ defimpl Collectable, for: Plug.Conn do
   def into(conn) do
     IO.warn(
       "using Enum.into/2 for conn is deprecated, use `Plug.Conn.chunk/2` " <>
-        "and `Enum.reduce_while/3` instead (see the Plug.Conn.chunk/2 docs for an example)"
+        "and `Enum.reduce_while/3` instead (see the `Plug.Conn.chunk/2` docs for an example)"
```

Comment:
I actually prefer to keep error messages and warnings without backticks. But, if we are including them here, I think the `Enum.into/2` above should also have them 🙂